### PR TITLE
Feature/additionsleadkitfix

### PR DESCRIPTION
**Pull request by NikAshanin**

### CODE REVIEW

File: LeadKit/LeadKit/Extensions/UIApplication/UIApplication+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+     static method changes root controller in window and sets status bar style
+     
+     - parameter controller:     new root controller
+     - parameter statusBarStyle: new status bar style
+     - parameter appWindow:      application window
+    */
+    public static func changeRootController(controller: UIViewController,
+                                            statusBarStyle: UIStatusBarStyle,
+                                            appWindow: UIWindow?) {
+        guard let window = appWindow else {
```
Comment (petropavel13) [inline LeadKit/LeadKit/Extensions/UIApplication/UIApplication+Extensions.swift]: instead create extension for UIWindow

File: LeadKit/LeadKit/Extensions/TableDirector/TableDirector+Extensions.swift [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+     method reloads section at index with animation
+     
+     - parameter index:     current reloaded section index
+     - parameter animation: reloading animation. Default .none
+
+     - returns: self
+     */
+    @discardableResult
+    public func reload(sectionAtIndex index: Int, with animation: UITableViewRowAnimation = .none) -> Self {
+        let action = { [tableView] in
```
Comment (petropavel13) [inline LeadKit/LeadKit/Extensions/TableDirector/TableDirector+Extensions.swift]: add guard for strong table view?

File: LeadKit/LeadKit/Extensions/TableDirector/TableDirector+Extensions.swift [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+     - returns: self
+     */
+    @discardableResult
+    public func replace(section: TableSection, atIndex index: Int, reload: Bool = true) -> Self {
+        if index < sections.count {
+            remove(sectionAt: index)
+        }
+        insert(section: section, atIndex: index)
+        if reload {
+            self.reload(sectionAtIndex: index)
```
Comment (petropavel13) [inline LeadKit/LeadKit/Extensions/TableDirector/TableDirector+Extensions.swift]: self is required?

Comment (NikAshanin) [inline LeadKit/LeadKit/Extensions/TableDirector/TableDirector+Extensions.swift]: yes, there is an error without it

**Pull request merged**